src/SFA.DAS.EmployerIncentives/Services/CommitmentsV2Service.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+using SFA.DAS.EmployerIncentives.Infrastructure.Api;
+using SFA.DAS.EmployerIncentives.Interfaces;
+
+namespace SFA.DAS.EmployerIncentives.Services
+{
+    public class CommitmentsV2Service : ICommitmentsV2Service
+    {
+        private readonly IRestApiClient _restApiClient;
+
+        public CommitmentsV2Service(HttpClient client)
+        {
+            _restApiClient = new RestApiClient(client);
```

Comment:
This doesn't seem right to me 👀

Reply:
This is due to the way we need to wire up the clients. If you look at how we add the HttpClients in AppStart\ServiceRegistrationExtensions.cs

You can see how we are adding clients.